includes/03-init.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * Plugin Initialization including assets registration for WordPress.
+ *
+ * @package ubc_wp_vote
+ */
+
+namespace UBC\CTLT\WPVote;
+
+// If this file is called directly, abort.
+if ( ! defined( 'ABSPATH' ) ) {
+	die;
+}
+
+add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\\load_styles_scripts' );
+add_action( 'wp-hybrid-clf_after_entry', __NAMESPACE__ . '\\render_post_content_actions' );
+add_action( 'wp-hybrid-clf_after_comment', __NAMESPACE__ . '\\render_comment_content_actions' );
+
+/**
+ * Enqueue application styles and scripts
+ *
+ * @return void
+ */
+function load_styles_scripts() {
+	global $post;
+
+	$valid_post_types = get_post_types(
+		array(
+			'public' => true,
+		)
+	);
+	$valid_post_types = array_filter(
+		$valid_post_types,
+		function( $post_type ) {
+			return ! in_array( $post_type, UBC_WP_VOTE_POST_TYPES_TO_EXCLUDE );
+		}
+	);
+
+	if ( is_singular() && in_array( $post->post_type, $valid_post_types ) ) {
```

Comment:
Not absolutely necessary, but I'd like us to try to use guards as much as possible. It keeps nested code to a minimum, and makes testing easier. I also feel it looks neater and is easier to read. So here's we do something like

```
if ( ! is_singular() || ! in_array( $post->post_type, $valid_post_types ) {
    return;
}

// enqueue styles/scripts here
```